queries.yaml

```diff
@@ -975,19 +958,77 @@ post_process:
   - fn: vectordatasource.transform.drop_properties_with_prefix
     params: {prefix: mz_}
 
+  # drop small inners in buildings before we do the merge.
+  - fn: vectordatasource.transform.drop_small_inners
+    params:
+      end_zoom: 15
+      source_layers: [buildings]
+      pixel_area: 0.25
+
   - fn: vectordatasource.transform.merge_building_features
     params:
       source_layer: buildings
       start_zoom: 13
-      end_zoom: 15
+      end_zoom: 16
       quantize:
-        13: vectordatasource.transform.quantize_height_round_nearest_10_meters
-        14: vectordatasource.transform.quantize_height_round_nearest_5_meters
+        13: vectordatasource.transform.quantize_height_round_nearest_20_meters
+        14: vectordatasource.transform.quantize_height_round_nearest_10_meters
+        15: vectordatasource.transform.quantize_height_round_nearest_10_meters
+      # todo: keep instead of drop?
       drop:
         - name
+        - short_name
         - addr_housenumber
         - addr_street
-      max_merged_features: 1000
+        - osm_relation
+      # NOTE: max_merged_features is set to keep the time taken for geometry
+      # merging down, as it seems to go up with the square of the number of
+      # features.
+      max_merged_features: 2000
+
+  # drop small inners in buildings after we do the merge, too! because the merge
+  # process may also have created small inners.
+  - fn: vectordatasource.transform.drop_small_inners
+    params:
+      end_zoom: 15
+      source_layers: [buildings]
+      pixel_area: 0.25
```

Comment:
Great idea!

Reply:
How is this different than line 962? We should only do this procedure once.